portfolio/src/main/webapp/community.html

```diff
@@ -20,22 +20,23 @@
             <div id="form">
                 <h2>Community Page</h2>
                 <form action="/comments" method="POST">
-                    <p>Username:</p>
+                    <label>Username:</label>
                     <input name="username" />
 
-                    <p>Comment</p>
+                    <label>Comment</label>
```

Comment:
Sorry to poke you about labels again =o
To gain the programmatic benefits that naturally connect labels to their inputs, the "for" attribute of the label must be set to a value equal to the value of the "id" attribute in the input.

So in these cases for the user name example, you'd want to add for="username" to the label and id="username" to the input.  And a similar change for the comment.  I know it can look weird for an input to have both an "id" and a "name" attribute with the same value (i.e. id="username" name="username"), but that is the standard way to do things!

Reply:
Added for's